src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Build/StartupProjectSingleTargetGlobalBuildPropertyProvider.cs

```diff
@@ -43,10 +46,8 @@ internal class StartupProjectSingleTargetGlobalBuildPropertyProvider : StaticGlo
             _projectSystemOptions = projectSystemOptions;
         }
 
-        public override async Task<IImmutableDictionary<string, string>> GetGlobalPropertiesAsync(CancellationToken cancellationToken)
+        public override Task<IImmutableDictionary<string, string>> GetGlobalPropertiesAsync(CancellationToken cancellationToken)
```

Comment:
Does it still need the `Async` suffix?

Reply:
Good question.

The `Async` suffix advertises to the caller that they should await the return value. We still return a `Task<>` here, so the `Async` suffix still applies.

The `async` modifier controls whether the method's implementation needs to compiler to produce a state machine. This method was changed to avoid any `await`s, meaning we don't need the `async` modifier.

Reply:
Oh, I see, thanks for the clarification!